content/tracing/advanced_usage/custom_tagging.md

```diff
@@ -44,6 +44,46 @@ class ServletImpl extends AbstractHttpServlet {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+
+**Adding tags to a span**
+
+You can add tags directly to a span by calling `set_tag`. For example with the following Flask route handler:
```

Comment:
`with the following route handler` -> let's remove Flask references from the docs, but let's keep it generic.